src/binding/c/lottieanimation_capi.cpp

```diff
@@ -205,4 +205,28 @@ lottie_animation_property_override(Lottie_Animation_S *animation,
     }
     va_end(prop);
 }
+
+LOT_EXPORT LOTMarkerList*
+lottie_animation_get_markerlist(const Lottie_Animation_S *animation)
+{
+   if (!animation) return nullptr;
+
+   LOTMarkerList *markerlist;
+   auto markers = animation->mAnimation->markers();
+   if (markers.size() == 0) {
+       return nullptr;
+   }
+
+   markerlist = new LOTMarkerList();
+   markerlist->size = 0;
+   markerlist->ptr = new LOTMarker[markers.size()]();
+
+   for(size_t i = 0; i < markers.size(); i++) {
+       markerlist->ptr[i].name= std::move(std::get<0>(markers[i]).c_str());
```

Comment:
std::move will not help here. as we are directly accessing the underline pointer.